TMDbLib/Objects/Exceptions/NotFoundException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMDbLib.Objects.Exceptions
+{
+    public class NotFoundException : TMDbHttpException
```

Comment:
We should probably not name our exceptions "Http". Sure. If something is http specific, go for it - but for the higher level stuff, like "not found", the inheritance tree should not include "Http" (or any other transport protocol).